packages/core/src/App/Constants/platform-config.js

```diff
@@ -17,20 +17,6 @@ const platform_config = [
         description: () => localize('Automated trading at your fingertips. No coding needed.'),
         link_to: routes.bot,
     },
-    {
```

Comment:
since we do not have these pages anymore , we do not need to redirect user to traders hub page when menually adding the mt5 url in the browser and we should show the error page instead.
![image](https://user-images.githubusercontent.com/82078941/225005024-6605c523-2980-4cb2-83c4-e964eff517a4.png)